content/applications/finance/accounting/taxes/vat_validation.rst

```diff
@@ -1,43 +1,44 @@
+`VAT Information Exchange System <https://ec.europa.eu/taxation_customs/vies/#/vat-validation>`_, or
+**VIES**, is a tool provided by the European Commission that allows you to check the validity of VAT
+numbers for companies registered in the European Union.
 
-Odoo provides a feature to **Verify VAT Numbers** when you save a contact. This helps you make sure
-that your contacts provided you with a valid VAT number without leaving Odoo interface.
+Odoo uses this tool to verify your contacts' VAT numbers directly from Odoo's interface.
 
-Configuration
-=============
+.. note::
+   Regardless of whether or not the Verify VAT Numbers feature is enabled, Odoo checks the format of
+   a contact's VAT against the expected format of VAT numbers from that country.
 
-To enable this feature, go to :menuselection:`Accounting --> Configuration --> Settings --> Taxes`,
-enable the **Verify VAT Numbers** feature, and click on *Save*.
-
-.. image:: vat_validation/vat-validation-configuration.png
-   :align: center
-   :alt: Enable "Verify VAT Numbers" in Odoo Accounting
-
-VAT Number validation
-=====================
+.. seealso::
+   `VAT identification number formats <https://en.wikipedia.org/wiki/VAT_identification_number>`_
 
-Whenever you create or modify a contact, make sure to fill out the **Country** and **VAT** fields.
+VIES VAT number validation
+==========================
 
-.. image:: vat_validation/vat-validation-contact-form.png
-   :align: center
-   :alt: Fill out the contact form with the country and VAT number before clicking on *Save*
+To activate this feature, go to the :guilabel:`Taxes` section in :menuselection:`Accounting -->
+Configuration --> Settings`, enable the :guilabel:`Verify VAT Numbers` feature, and click on
+:guilabel:`Save`.
 
-When you click on *Save*, Odoo runs a VIES VAT number check, and displays an error message if the
-VAT number is invalid.
+Once the :guilabel:`Verify VAT Numbers` feature is enabled, if the :guilabel:`Tax ID` field is
+populated *and* the contact's country is different from your company's country, Odoo displays an
+:guilabel:`Intra-Community Valid` checkbox. Odoo tests the VAT number through the VIES and
+automatically checks or unchecks the :guilabel:`Intra-Community Valid` checkbox depending on the
+validity of the VAT number.
 
-.. image:: vat_validation/vat-validation-error.png
-   :align: center
-   :alt: Odoo displays an error message instead of saving when the VAT number is invalid
+.. image:: vat_validation/intra-community-valid.png
+   :alt: Intra-community valid checkbox on the contact record
 
-.. important::
-   This tool checks the VAT number's validity but does not check the other fields' validity.
+.. warning::
+   While it is possible to manually change the :guilabel:`Intra-Community Valid` field on a contact,
+   this should only be done by accountants who are aware of and accept the consequences of using an
+   invalid VAT. This change is logged in the chatter for transparency.
```

Comment:
I would phrase it differently, as it is never okay to use an invalid VAT number, even if you are ready to face the consequences ^^ The warning should rather positively say that it is expected from businesses that they only issue intra-community invoices to businesses with verified VAT numbers.

Also, transparency isn't the only reason there is that checkbox: in my understanding, it is indeed to manually "force" validate a number when the automatic verification isn't correct. For example, your supplier does not have a valid VAT number anymore, but it was still valid at the time of issuance of their invoice, and you still need to encode it in your accounting.

Reply:
Per the Accounta Matata task:

> In both cases, the accountants must be able in back office to override the output of the VIES check. It can happen the VIES service is unavailable. It can happen partner's whose companies being recently created are not yet available in the VIES service. By allowing the accountants to take control on this checkbox, no flow is ever permanently blocked.  This is also why the checkbox is labelled "Valid" rather than "VIES valid".  

https://www.odoo.com/web#id=3218194&cids=1&menu_id=4720&action=333&active_id=967&model=project.task&view_type=form

Is your issue with the term "invalid VAT" or something else? I could change it to "valid but unverified VAT" - I think that would more precisely explain that the VAT is valid, but the VIES check doesn't automatically verify it.

The line "This change is logged in the chatter for transparency" is referring to a manual change of the Intra-Community Valid checkbox. So you're right that the checkbox isn't only there for transparency, but transparency is the reason that checking/unchecking that field is logged in the chatter.

Let me know what you think :)